src/Filter/QueryParameterValidateListener.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Filter;
```

Comment:
Please move it in the `ApiPlatform\EventListener` namespace.
What about `ValidateQueryParametersListener` for the name?

Reply:
It's okay if it's refactored later as said in https://github.com/api-platform/core/pull/1692#issuecomment-391711723 I guess?